packages/frontend2/src/app/(new)/(other)/scaling-next/summary/page.tsx

```diff
@@ -20,11 +21,44 @@ export const metadata = getDefaultMetadata({
 
 export default async function Page() {
   const tvl = await getTvl()
+  const hourly = (await getTvlChart({ range: '7d', type: 'layer2' })).map(
+    ([timestamp, native, canonical, external, ethPrice]) =>
+      [
+        timestamp,
+        (native + canonical + external) / 100,
+        native / 100,
+        canonical / 100,
+        external / 100,
+        (native + canonical + external) / ethPrice / 100,
+        native / ethPrice / 100,
+        canonical / ethPrice / 100,
+        external / ethPrice / 100,
+      ] as [
+        number,
+        number,
+        number,
+        number,
+        number,
+        number,
+        number,
+        number,
+        number,
+      ],
+  )
   const { layer2s, layer3s } = await getScalingSummaryEntries(tvl)
 
   return (
     <div className="mb-20">
-      <TvlChart data={tvl.layers2s} milestones={HOMEPAGE_MILESTONES} />
+      <TvlChart
+        data={{
+          ...tvl.layers2s,
+          hourly: {
+            ...tvl.layers2s.hourly,
+            data: hourly,
+          },
+        }}
+        milestones={HOMEPAGE_MILESTONES}
+      />
```

Comment:
Same as above, why do we need to construct such an override explicitly?

Reply:
This and the above cast is only left for dev purposes. To not add any more complexity I'll do it as part of L2B-6115.

Reply:
Anyway - I reverted those changes here.